app/test_main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+from unittest import mock
+import pytest
+from app.main import can_access_google_page
+from typing import Callable
+
+
+@pytest.fixture()
+def mocked_valid_google_url() -> Callable:
+    with mock.patch("app.main.valid_google_url") as mock_valid_google_url:
```

Comment:
You can use decorators instead. And add a fixture to avoid writing them with every test